index.js

```diff
@@ -100,14 +138,16 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
-
-// console.log(myMap([1,2,3,4,5], triple)) 
+console.log("%cCallbacks - Question 1", "color: red")
+function triple(myMap) {
+  console.log(myMap * 3)
+}
+ 
```

Comment:
This looks like it'll work. However, this function is being passed in a number (not a callback). This function should solely take in a number and return it tripled.